### ISSUE TITLE
Could not move Obsidian's window after its v1.0

### ISSUE THREAD

**Issue by vrtmrz** [vrtmrz/ninja-cursor#5]
Obsidian ignores dragging while ninja-cursor is enabled.

```
SYSTEM INFO:
	Obsidian version: v1.0.0
	Installer version: v0.14.6
	Operating system: Windows 10 Pro 10.0.19044
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 24
	Plugins enabled: 7
		1: Self-hosted LiveSync v0.16.0
		2: Obsidian42 - BRAT v0.6.35
		3: TagFolder v0.15.5
		4: Workaround keyboard v0.0.5
		5: Calendar v1.5.10
		6: Screwdriver v0.0.3
		7: Templater v1.14.3

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

```

**Comment by vrtmrz** [comment]
Fixed at 0.0.8.

**Comment by vrtmrz** [comment]
Fixed at 0.0.8.
